packages/web/src/components/basic/NumberBox.js

```diff
@@ -209,6 +218,7 @@ NumberBox.propTypes = {
 	selectedValue: types.selectedValue,
 	setQueryOptions: types.funcRequired,
 	setCustomQuery: types.funcRequired,
+	enableAppbase: types.bool,
```

Comment:
@anik-ghosh-au7 Use `VueTypes` instead of `types` for boolean props. There are some issues that would set the default value to `true`. Make that change at all place.

Reply:
@bietkul made the changes